### ISSUE TITLE
`pytest -v tests\plotting` raises the error `tests/plotting/test_plotting.py::test_import_gltf Fatal Python error: Segmentation fault`.

### ISSUE THREAD

**Issue by tkoyama010** [pyvista/pyvista#1769]
@tkoyama010 I have a question regarding local testing: I tried to run the automated test on a local virtual Ubuntu 20.04. But `pytest -v tests\plotting` raises the error `tests/plotting/test_plotting.py::test_import_gltf Fatal Python error: Segmentation fault`. When I tried to use `act` to run the GitHub action in the original environment, I got the errors like `[Unit Testing and Deployment/Linux Unit Testing-3 ]   ❗  ::error::Version 3.8 with arch x64 not found%0AAvailable versions:%0A%0A`.
Is there a guide how to setup a testing environment under a virtual linux?

_Originally posted by @beppo-dd in https://github.com/pyvista/pyvista/pull/1750#discussion_r738295673_

**Comment by beppo-dd** [comment]
This error occurs when runnig Ubuntu 20.04, 64 bit (x86_64, 2 CPUs, 3.8 GiB RAM, 128 VRAM) in Virtualbox 6.1 (under Windows host), with Python 3.8.10 (Packages: pyvista 0.32.dev0, vtk 9.0.3, numpy 1.21.3, imageio 2.10.1, appdirs 1.4.4, scooby 0.5.7, meshio 4.4.6, matplotlib 3.4.3). I installed the Virtualbox Guest Additions and enabled 3D acceleration for the guest system. I used already https://dev.pyvista.org/extras/developer_notes.html#testing to setup this system.
